operator/PROJECT

```diff
@@ -2,6 +2,6 @@ domain: keda.sh
 repo: github.com/kedacore/http-add-on
 resources:
 - group: http
-  kind: ScaledObject
+  kind: HTTPHTTPScaledObject
```

Comment:
Should this just be `HttpScaledObject`?

Reply:
Yep, it should. I think it was a rename issue. I'll fix it!